CHANGELOG.md

```diff
@@ -32,6 +162,24 @@ BUG FIXES:
 * Fix incompatibility with Helm 3.1.2. [[GH-390](https://github.com/hashicorp/consul-helm/issues/390)]
 * Ensure the Consul Enterprise license gets applied, even if servers take a long time to come up. [[GH-348](https://github.com/hashicorp/consul-helm/pull/348))
 
+DEPRECATIONS:
```

Comment:
Is this for 0.18.0? or just weird git diff?

Reply:
ack, thanks. Meant for this to be under upcoming.